openapi_specgen/schema.py

```diff
@@ -95,6 +98,15 @@ def resolve_any(openapi_schema_resolver: "OpenApiSchemaResolver", data_type: typ
     return {"$ref": openapi_schema_resolver.get_component_ref("AnyValue")}
 
 
+def resolve_enum(openapi_schema_resolver: "OpenApiSchemaResolver", data_type: type):
+    if not isinstance(data_type, enum.EnumMeta):
+        return
+
+    openapi_type = OPENAPI_TYPE_MAP.get(data_type.__base__)
+
+    return {"type": openapi_type}
```

Comment:
```suggestion
def resolve_enum(openapi_schema_resolver: "OpenApiSchemaResolver", data_type: type):
    if not isinstance(data_type, enum.EnumMeta):
        return

    openapi_schema = openapi_schema_resolver.get_schema(data_type.__base__)
    openapi_schema["enum"] = [member.value for member in data_type]

    return openapi_schema
```

if you can apply these changes I'll merge, test and release